ext/flogo/trigger/gorillamuxtrigger/trigger.json

```diff
@@ -65,6 +65,34 @@
     {
       "name": "basicAuthFile",
       "type": "string"
+    },
+    {
```

Comment:
in the future, it may be better to have a single setting of type object for things like ldap settings. that way you don't pollute the trigger with so many optional settings.